server/src/domain/person/person.dto.ts

```diff
@@ -99,18 +105,18 @@ export class AssetFaceResponseDto extends AssetFaceWithoutPersonResponseDto {
   person!: PersonResponseDto | null;
 }
 
+export class FaceDto {
+  @ValidateUUID()
+  id!: string;
+}
```

Comment:
We're `PUT`-ing to a route with an id and the dto is an id but they are different types. I think it might make more sense for this to be `personId`, which is what I believe it actually is. If this is a personId then the dto should probably be called ReassignFaceDto or something